CHANGELOG.md

```diff
@@ -2007,4 +2008,4 @@ First release of OpenTelemetry Collector Contrib.
 [v0.2.7]: https://github.com/open-telemetry/opentelemetry-collector-contrib/compare/v0.2.6...v0.2.7
 [v0.2.6]: https://github.com/open-telemetry/opentelemetry-collector-contrib/compare/v0.0.5...v0.2.6
 [v0.0.5]: https://github.com/open-telemetry/opentelemetry-collector-contrib/compare/v0.0.1...v0.0.5
-[v0.0.1]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.0.1
\ No newline at end of file
+[v0.0.1]: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/v0.0.1
```

Comment:
Hmm, saw the same thing in some other PR, realized there seems to be a trailing space added by https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/8548

Reply:
I'm not blocking this PR because of this :-) Merging.